app/Nova/User.php

```diff
@@ -112,6 +113,10 @@ class User extends Resource
     public function fields(NovaRequest $request): array
     {
         return [
+            Hidden::make('Has Ever Logged In')
+                ->showOnCreating()
+                ->default(static fn (Request $r) => 0),
```

Comment:
Nitpick - this should be a boolean attribute
```suggestion
                ->default(static fn (Request $r): false => false),
```